docs/build-and-push-assets.md

```diff
@@ -86,11 +86,12 @@ This is not the "simplest" possible example, but it showcases all the recommenda
 
 ## Secrets
 
-| Name                 | Description                                  |
-| -------------------- | -------------------------------------------- |
-| `NPM_REGISTRY_TOKEN` | Authentication for the private npm registry. |
-| `GITHUB_USER_EMAIL`  | GitHub User email used in `git config`.      |
-| `GITHUB_USER_NAME`   | GitHub User name used in `git config`.       |
+| Name                  | Description                                                          |
+|-----------------------|----------------------------------------------------------------------|
+| `NPM_REGISTRY_TOKEN`  | Authentication for the private npm registry.                         |
+| `GITHUB_USER_EMAIL`   | GitHub User email used in `git config`.                              |
+| `GITHUB_USER_NAME`    | GitHub User name used in `git config`.                               |
+| `GITHUB_USER_SSH_KEY` | Private SSH key associated to the GitHub user in "GITHUB_USER_NAME". |
```

Comment:
```suggestion
| Name                  | Description                                                                   |
|-----------------------|-------------------------------------------------------------------------------|
| `NPM_REGISTRY_TOKEN`  | Authentication for the private npm registry.                                  |
| `GITHUB_USER_EMAIL`   | GitHub User email used in `git config`.                                       |
| `GITHUB_USER_NAME`    | GitHub User name used in `git config`.                                        |
| `GITHUB_USER_SSH_KEY` | Private SSH key associated with the GitHub user passed as `GITHUB_USER_NAME`. |
```

Reply:
Done in [3855f95](https://github.com/inpsyde/reusable-workflows/pull/12/commits/3855f95897238b6b52c751cc2b284ef522b79e9c)